hack/onboarding.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+if [ $# -le 1 ] || [ "$#" == "--help" ] || [ "$#" == "-h" ]; then
+	echo "Populate the namespace resource definition."
+	echo -e "\nUsage: $0 namespace owner-team [description]\n"
+	exit 0
+fi
+
+APP_NAME="cluster-scope"
+NAMESPACE=$1
+OWNER=$2
+DESCRIPTION=$3
+
+create_namespace() {
+	mkdir -p $APP_NAME/base/namespaces/$NAMESPACE
+	cat <<-EOF > $APP_NAME/base/namespaces/$NAMESPACE/namespace.yaml
+	---
+	apiVersion: v1
+	kind: Namespace
+	metadata:
+	  annotations:
+	    openshift.io/display-name: "$DESCRIPTION"
+	    openshift.io/requester: $OWNER
+	  name: $NAMESPACE
+	spec: {}
+	EOF
+	cat <<-EOF > $APP_NAME/base/namespaces/$NAMESPACE/kustomization.yaml
+	---
+	apiVersion: kustomize.config.k8s.io/v1beta1
+	kind: Kustomization
+
+	namespace: $NAMESPACE
+
+	resources:
+	- namespace.yaml
+
+	components:
+	- ../../../components/project-admin-rolebindings/$OWNER
+	EOF
+	echo "Namespace base created at '$APP_NAME/base/namespaces/$NAMESPACE'."
+}
+
+create_project-admin-rolebinding() {
```

Comment:
so afaik hyphens (i.e. `-`) in `names` are a bash thing, not posix -- so this could break on some systems, can we rename this to `create_project_admin_rolebinding`, and the function call below as well.